runtime/cudaq/CMakeLists.txt

```diff
@@ -19,7 +19,8 @@ add_library(${LIBRARY_NAME}
                 algorithms/draw.cpp
                 qis/state.cpp
                 qis/remote_state.cpp
-                platform/quantum_platform.cpp 
+                platform/quantum_platform.cpp
+                qis/execution_manager_c_api.cpp
```

Comment:
nit: it would be preferable to keep the `qis` items together if possible (i.e. move the new `platform` one above the other `qis` entries).

Reply:
Fixed in aebe2ece6a061854947bd784413a77ee55440461